.github/workflows/get-tools-new-versions.yml

```diff
@@ -16,17 +16,40 @@ jobs:
         tool:
           - name: 'Xamarin'
             image: 'https://avatars.githubusercontent.com/u/790012?s=200&v=4'
+            releases-url: 'null'
+            filter-parameter: 'null'
+            filter-arch: 'null'
           - name: 'Python'
             image: 'https://avatars.githubusercontent.com/u/1525981?s=200&v=4'
+            releases-url: 'https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json'
+            filter-parameter: 'version'
+            filter-arch: 'x64'
           - name: 'PyPy'
             image: 'https://avatars.githubusercontent.com/u/318667?s=200&v=4'
+            releases-url: 'https://downloads.python.org/pypy/versions.json'
+            filter-parameter: 'python_version'
+            filter-arch: 'x86'
+          - name: 'Node'
+            image: 'https://avatars.githubusercontent.com/u/9950313?s=200&v=4'
+            releases-url: 'https://raw.githubusercontent.com/actions/node-versions/main/versions-manifest.json'
+            filter-parameter: 'version'
+            filter-arch: 'x64'
+          - name: 'Go'
+            image: 'https://avatars.githubusercontent.com/u/4314092?s=200&v=4'
+            releases-url: 'https://raw.githubusercontent.com/actions/go-versions/main/versions-manifest.json'
+            filter-parameter: 'version'
+            filter-arch: 'x64'
     name: 'Searching for new versions of ${{ matrix.tool.name }}'
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
What is the reason to use windows?

Reply:
Previously it was changed to get rid of "Where-Object: Cannot convert value "16" to type "System.Version". Error: "Version string portion was too short or too long." with Node sorting. It worked locally with PS 7, but was failing in the workflow.
Currently it just left with windows after all the debugging.

Reply:
Where at that line do we get 16?

Reply:
Cached "node" versions are specified in the toolset with just major version opposed to other tools with minor versions. So in the line 21 of "`verify-common-tool-parser.psm1`" we will get just 16 with `$latestVersion.TrimEnd(".*")` which causing error in comparison.

Reply:
Now we don't have this statement. I think we can return ubuntu.

Reply:
Indeed

Reply:
Updated